code/style.css

```diff
@@ -1,4 +1,282 @@
+/* Style for Project 2, Technigo Bootcamp: News site */
+
+* {
+  box-sizing: border-box;
+}
+
 html {
-  background: #0025ff;
-  color: #fff;
+  scroll-behavior: smooth;
+  background: #ffffff;
+  color: #000000,
+}
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  line-height: 1.7;
+	margin: 0;
+}
+
+.wrapper {
+padding: 1rem;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'Alata', sans-serif;
+}
+
+p {
+  font-size: 0.95em;
+}
+
+a {
+	text-decoration: none;
+}
+
+/* Style for header/navbar and logo */
+header {
+  background-color: #EFE1D6;
+  width: 100%;
+  text-align: center;
+  box-shadow: 12px 5px 10px 1px #dcdcdc;
+}
+
+.logo {
+  width: 80px;
+}
+
+.main-nav, .navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.main-nav {
+	margin-top: 5px;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.logo a,
+.main-nav a {
+	padding: 10px 15px;
+	text-transform: uppercase;
+	text-align: center;
+	display: block;
+}
+
+.main-nav a {
+	color: #000000;
+  font-size: .8em;
+  font-weight: lighter;
+}
+
+.main-nav a:hover {
+  color: #ac7675;
+  transition: all .3s;
+}
+
+/* Style for the big-news section */ 
+.big-news {
+  /* Photo by Jakub Dziubak on Unsplash */
+  background-image: url("img/jakub-dziubak-XtUd5SiX464-unsplash.jpg");
+  background-size: cover;
```

Comment:
Bravo!! If you want the image to show the coffee cup on smaller devices also, you check out this CSS property:
https://cssreference.io/property/background-position/